src/main/scala/zio/s3/Live.scala

```diff
@@ -237,4 +235,7 @@ object Live {
       ()
     }
   }
+
+  final val SingleEmptyChunkStream = ZStream(Chunk.empty)
```

Comment:
dont need to extract

Reply:
https://github.com/zio/zio-s3/pull/117/commits/a1067926d148b8d2bf79a520fa5771945a73510c#diff-4371b13eba4391eed947c00dddac1f12L239